### ISSUE TITLE
[Small docs request] Clarify support for Express 4/5

### ISSUE THREAD

**Issue by webketje** [twigjs/twig.js#855]
In the current README docs, it is said that "Twig can be used in Express 2 and 3".
But the most important is whether it can be used with Express 4 and 5...

**Comment by willrowe** [comment]
I do not believe the tests cover those versions. If you would like to submit a PR that adds tests to confirm compatibility, then the docs can certainly be updated.
